package.json

```diff
@@ -119,7 +119,7 @@
         }
     },
     "engines": {
-        "node": ">=14.0.0 <15.0.0",
+        "node": ">=18.16.0 <19.0.0",
```

Comment:
should we just remove the upper restriction and do something like?
`"node": ">=14.x",` or `"node": ">=16.x",`

this way it doesn't force apps to immediately upgrade and parent apps can try to use v20 if they want

Reply:
how would this work if the use of that legacy openssl flag is only available in 17 and above? would parent apps still be required to immediately upgrade?

the openssl flag is only invoked in local dev or build using webpack, but i dont know what impact that has on parent apps.

Reply:
depends on if versions lower than node v17 will ignore flags they don't recognize

Reply:
BUIE certainly doesn't build when the flag is included on node 14, but in a parent app, they're not building buie right? or do they, when the whole parent app is built?